### DISCUSSION TITLE
Welcome to Eduscope Mods Discussions!

### DISCUSSION THREAD

**Discussion by notnavindu** [notnavindu/SLIIT-Eduscope-Mods#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Amishkaz** [comment]
The old way to set the speed was easy in addition to the slider if u can add set values like x1 x1.5 it will be easy

**Reply by notnavindu** [reply]
Thanks for the feedback. I will add those back in soon
